### PR TITLE
Use reconciler-test infra, drop rigging.

### PR DESCRIPTION
**Pull request by vaikas**
# Changes

-
-
-

/kind <kind>

Fixes #

**Release Note**

```release-note
:broom: Use reconciler-test infra for e2e tests, drop rigging.
```

**Docs**

```docs

```

### CODE REVIEW

File: test/e2e/config/direct/direct.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+	"knative.dev/reconciler-test/pkg/environment"
+	"knative.dev/reconciler-test/pkg/feature"
+	"knative.dev/reconciler-test/pkg/manifest"
+)
+
+func init() {
+	environment.RegisterPackage(manifest.ImagesLocalYaml()...)
+	environment.WithImages(map[string]string{
+		"ko://knative.dev/eventing-rabbitmq/test/e2e/cmd/producer": "kind.local/producer-35be09e7925e4d603eb8a2e5a9207882:891371837a579fe86e52734e79899343109b38419a4a2d56e88ee35057163d8e",
```
Comment (n3wscott) [inline test/e2e/config/direct/direct.go:33]: Might want to add a `// DO NOT SUBMIT` here

File: test/e2e/config/direct/direct.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	})
+}
+
+func Install() feature.StepFn {
+	return func(ctx context.Context, t *testing.T) {
+		if _, err := manifest.InstallLocalYaml(ctx, map[string]interface{}{"producerCount": 5}); err != nil {
```
Comment (n3wscott) [inline test/e2e/config/direct/direct.go:39]: I would pass `producerCount` in as a param on `Install()`

File: test/e2e/config/direct/producer.yaml [hunk 3]
```diff
@@ -28,7 +28,7 @@ spec:
     spec:
       containers:
         - name: producer
-          image: {{ .images.producer }}
+          image: ko://knative.dev/eventing-rabbitmq/test/e2e/cmd/producer
```
Comment (n3wscott) [inline test/e2e/config/direct/producer.yaml:31]: yeah, I think this is better. Less magical.

File: test/e2e/smoke_test_broker.go [hunk 4]
```diff
@@ -0,0 +1,35 @@
+// +build e2e
```
Comment (n3wscott) [inline test/e2e/smoke_test_broker.go:1]: I would drop this tag.

File: test/e2e/smoke_test_broker_trigger.go [hunk 5]
```diff
@@ -0,0 +1,35 @@
+// +build e2e
```
Comment (n3wscott) [inline test/e2e/smoke_test_broker_trigger.go:1]: and drop this one

File: test/e2e/config/direct/direct.go (hunk 1 above)
Comment (vaikas) [reply test/e2e/config/direct/direct.go]: removed.

**Review by n3wscott — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**